plugin/kubernetes/kubernetes.go

```diff
@@ -40,7 +40,7 @@ type Kubernetes struct {
 	Namespaces       map[string]bool
 	podMode          string
 	endpointNameMode bool
-	Fallthrough      bool
+	Fallthrough      *[]string // nil = disabled, empty = all zones, o/w zones
```

Comment:
a slice is already a pointer, so a pointer to a slice looks weird, unless you're doing something advanced/special?

Reply:
The zero value for the slice is `[]string{}` not `nil`. I need three states: nil, empty, populated. Or I need two separate fields.